eng/pipelines/merge-docs-main-to-live.yml

```diff
@@ -0,0 +1,88 @@
+parameters:
+- name: WhatIfPreference
+  type: boolean
+  default: false
+
+- name: DocsRepos
+  type: object
+  default:
+    - RepoName: azure-docs-sdk-java
+      RepoOwner: Azure
+      # Plan to move the allowlist to docs repo. Allowlist itself does not allow to change for auto-merge. 
+      # By having the allowlist in docs repo, we can unblock things quickly.
+      AllowlistPath: eng/pipelines/templates/scripts/docsms-allowlist/java-allowlist.txt
+    - RepoName: azure-docs-sdk-python
+      RepoOwner: MicrosoftDocs
+      AllowlistPath: eng/pipelines/templates/scripts/docsms-allowlist/python-allowlist.txt
+    - RepoName: azure-docs-sdk-node
+      RepoOwner: MicrosoftDocs
+      AllowlistPath: eng/pipelines/templates/scripts/docsms-allowlist/js-allowlist.txt
+
+- name: TargetBranch
+  type: string
+  default: live
+
+jobs:
+  - job: SyncUp
+    pool:
+      vmImage: ubuntu 20.04
+    variables:
+      DocRepoLocation: $(Pipeline.Workspace)/docs
+    steps:
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+      - ${{ each repo in parameters.DocsRepos }}:
+        - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          parameters:
+            SkipDefaultCheckout: true
+            Repositories:
+            - Name: ${{ repo.RepoOwner }}/${{ repo.RepoName }}
+              WorkingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
+
+        - task: Powershell@2
+          inputs:
+            pwsh: true
+            filePath: eng/pipelines/templates/scripts/create-tag-for-current-branch.ps1
+            arguments: 
+              -GithubUrl "https://$(azuresdk-github-pat)@github.com/${{repo.RepoOwner}}/${{repo.RepoName}}.git" `
+              -WhatIf:$${{ parameters.WhatIfPreference }}
+            workingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
+          displayName: Create tag for base branch
```

Comment:
I am thinking to do live replacement, try to perform some tests in local before go ahead. So it will be a concern anymore. If the live branch break again. I will continue work on this.